Docs/wallet.md

```diff
@@ -47,34 +47,34 @@ Create a Wallet from either a private key or custom signing functions
 #### wallet.provider ⇒ <code>NahmiiProvider</code>
 The Nahmii Provider used by this wallet instance.
 
-**Kind**: instance property of [<code>Wallet</code>](#exp_module_nahmii-sdk--Wallet)  
+**Kind**: instance property of [<code>Wallet</code>](#exp_module_nahmii-sdk--Wallet)
 <a name="module_nahmii-sdk--Wallet+address"></a>
 
 #### wallet.address ⇒ <code>String</code>
 Returns the address for this wallet, required by ethers Wallet methods.
 
-**Kind**: instance property of [<code>Wallet</code>](#exp_module_nahmii-sdk--Wallet)  
+**Kind**: instance property of [<code>Wallet</code>](#exp_module_nahmii-sdk--Wallet)
 <a name="module_nahmii-sdk--Wallet+signerKey"></a>
 
 #### wallet.signerKey ⇒ <code>ethers.SignerKey</code> \| <code>undefined</code>
 If used with software wallet, returns an object containing signer related
 information and logic such as the private key, otherwise undefined
 
-**Kind**: instance property of [<code>Wallet</code>](#exp_module_nahmii-sdk--Wallet)  
-**Returns**: <code>ethers.SignerKey</code> \| <code>undefined</code> - The private key or undefined  
+**Kind**: instance property of [<code>Wallet</code>](#exp_module_nahmii-sdk--Wallet)
+**Returns**: <code>ethers.SignerKey</code> \| <code>undefined</code> - The private key or undefined
 <a name="module_nahmii-sdk--Wallet+getNahmiiBalance"></a>
 
 #### wallet.getNahmiiBalance() ⇒ <code>Promise</code>
-Retrieves nahmii balance for current wallet.
+Retrieves available nahmii balance for current wallet.
```

Comment:
Dont ever update the MD files inside the Docs folder manually. All changes should be done in JSDOC style comments inside the code. These files are auto generated by running "npm run build"